ReactiveCocoa/NSObject+RACDescription.m

```diff
@@ -48,3 +36,15 @@ - (NSString *)rac_description {
 }
 
 @end
+
+NSString *RACDescription(id object) {
+	if (getenv("RAC_DEBUG_SIGNAL_NAMES") != NULL) {
+		if ([object respondsToSelector:@selector(rac_description)]) {
+			return [object rac_description];
+		} else {
+			return [[NSString alloc] initWithFormat:@"<%@: %p>", object.class, object];
```

Comment:
This causes a compiler error "Property '``class``' not found on object of type '``__string id``'.
Do ``id`` objects definitely guarantee to implement this?